drivers/watchdog/wdt_sam0.c

```diff
@@ -0,0 +1,156 @@
+/*
+ * Copyright (c) 2017 Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <watchdog.h>
+
+#define WDT_REGS ((Wdt *)CONFIG_WDT_SAM0_BASE_ADDRESS)
+
+struct wdt_sam0_dev_data {
+	void (*cb)(struct device *dev);
+};
+
+static struct device DEVICE_NAME_GET(wdt_sam0);
+
+static void wdt_sam0_wait_synchronization(void)
```

Comment:
not sure if making this 'static inline' makes sense.

Reply:
I had a quick play.  Given that this function is only used in initialization, speed doesn't matter so it's down to code size.  Interestingly:

* At `-Os`, hello_world is 7444 bytes
* At `-Os` with inline, it's 7444 bytes
* At `-Os` with ALWAYS_INLINE, it's 7448 bytes :)

So I'd like to leave it as-is.